bigbluebutton-tests/playwright/user/multiusers.js

```diff
@@ -219,6 +219,44 @@ class MultiUsers {
     await this.userPage.hasText(`:nth-match(${e.dropdownWebcamButton}, 2)`, this.userPage.username, 'should display the username of Attendee on the second pinned webcam for Attendee');
   }
 
+  async focusUnfocusWebcam() {
+    await this.modPage.shareWebcam();
+    await this.userPage2.shareWebcam();
+
+    await this.modPage.getLocator(e.dropdownWebcamButton)
+      .filter({ hasText: this.userPage2.username })
+      .click();
+
+    await this.modPage.wasRemoved(e.focusWebcamBtn, 'should not display the focus webcam button for the moderator');
+
+    await this.userPage.shareWebcam();
+
+    await this.modPage.getLocator(e.dropdownWebcamButton)
+      .filter({ hasText: this.userPage2.username })
+      .click();
+    await this.modPage.getVisibleLocator(e.focusWebcamBtn).click();
+
+    await this.modPage.hasText(`:nth-match(${e.dropdownWebcamButton}, 1)`, this.userPage2.username, 'should display the username of User2 on the focused webcam for Mod');
+    await this.userPage2.hasText(`:nth-match(${e.dropdownWebcamButton}, 1)`, this.userPage2.username, 'should display the username of User2 on the first webcam for User1');
+
+    await this.userPage.getLocator(e.dropdownWebcamButton)
+      .filter({ hasText: this.userPage2.username })
+      .click();
+    await this.userPage.getVisibleLocator(e.focusWebcamBtn).click();
+
+    await this.userPage.hasText(`:nth-match(${e.dropdownWebcamButton}, 1)`, this.userPage2.username, 'should display the username of User1 on the focused webcam for User2');
+
+    await this.modPage.getLocator(e.dropdownWebcamButton)
+      .filter({ hasText: this.userPage2.username })
+      .click();
+    await this.modPage.getVisibleLocator(e.focusWebcamBtn).click(); // Unfocus webcam is the same button as focus webcam
```

Comment:
It's better to keep different selectors for the button (focus/unfocus) to avoid confusion

in the `/video-provider/video-list/video-list-item/user-actions/component.tsx:197`, you can dynamically set the data-test by checking the `focused` variable:

```ts
dataTest: focused ? 'unfocusWebcamBtn' : 'focusWebcamBtn',
```